packages/ruby/lib/readme/metrics.rb

```diff
@@ -8,9 +8,12 @@ class Metrics
     SDK_NAME = "Readme.io Ruby SDK"
     ENDPOINT = "https://metrics.readme.io/v1/request"
 
-    def initialize(app, api_key, &get_user_info)
+    def initialize(app, options, &get_user_info)
+      raise("Missing development flag") if options[:development].nil?
+
       @app = app
-      @api_key = api_key
+      @api_key = options[:api_key] || raise("Missing API key")
```

Comment:
Is it safe to raise? We don't want to crash requests at runtime

Reply:
I believe it's safe to raise. This should raise on application boot before any requests have a chance to run.